layouts/default.html

```diff
@@ -19,7 +19,24 @@
     <div id="header-wrapper">
       <div id="header">
         <div>
-          <a class="logo" href="/"><img src="/images/logo.png" alt="SeeClickFix API" /></a>
+          <a class="logo" href="/">
+            <svg height="33" viewBox="0 0 220 54" fill="none" role="img" xmlns="http://www.w3.org/2000/svg" aria-label="SeeClickFix Logo">
```

Comment:
Any thoughts or feedback on the `aria-label`: "SeeClickFix Logo"

Reply:
I would have used the <img tag and the `alt` attribute

I would use the text in the image, "CivicPlus SeeClickFix 311 CRM"